src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,26 @@
+const jwtService = require('../services/jwt.service.js');
+
+const authMiddleware = (req, res) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  const userData = jwtService.verify(token);
+
+  if (!userData) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  req.user = userData;
```

Comment:
After setting `req.user`, you should call `next();` to pass control to the next middleware or route handler.